### ISSUE TITLE
Doesn't close file after copy

### ISSUE THREAD

**Issue by ethanfrey** [homedepot/flop#17]
First of all, thank you for this library, it goes a few steps beyond ioutil and I was getting tired of writing a bunch of helper setup code to run test code that modifies the file system, especially high-level testing of binaries.

However, I got a problem... I copied a directory with some executables as part of a test setup code. When I tried to run one of the copied executables, I got:

```
Received unexpected error:
fork/exec /tmp/upgrade-manager-test484338219/upgrade_manager/genesis/bin/dummyd: text file busy
```

After digging into some issues, this seems to happen if it was writen, but not closed, and digging into the code, this seems to be the case if the Atomic option was not set. I think you need a `dstFile.Close()` here: https://github.com/homedepot/flop/blob/master/copy.go#L288

Current work-around is to pass in `flop.Options{Atomic: true}` and this will produce a closed file and no such error.

**Comment by swtch1** [comment]
Thank you for the submission.  The information regarding the option to nix the issue is very helpful.  I'll take a look.

**Comment by swtch1** [comment]
I didn't notice initially that you pointed to exactly what needed to change.  Thanks for that.  
This has been corrected in PR #18, and is now live in v0.1.5.

**Comment by ethanfrey** [comment]
Nice PR. Thanks.
